site/docs/v1/tech/apis/_tenant-response-body-base.adoc

```diff
@@ -326,6 +326,40 @@ The password minimum age in seconds. When enabled FusionAuth will not allow a pa
 [field]#{base_field_name}.minimumPasswordAge.enabled# [type]#[Boolean]# [since]#Available since 1.8.0#::
 Indicates that the minimum password age is enabled and being enforced.
 
+[field]#{base_field_name}.multiFactorConfiguration.authenticator.algorithm# [type]#[String]# [since]#Available since 1.26.0#::
+The algorithm used by the TOTP authenticator. The possible values are:
++
+* `HmacSHA1`
+* `HmacSHA256`
+* `HmacSHA512`
+
```

Comment:
For now we should probably just document this as a read only field that will always be `HmacSHA1`. We don't currently allow it to be modified. 

Same goes for `codeLength` and  `timeStep`.